src/main/mw.c

```diff
@@ -91,7 +92,8 @@ enum {
 /* VBAT monitoring interval (in microseconds) - 1s*/
 #define VBATINTERVAL (6 * 3500)       
 /* IBat monitoring interval (in microseconds) - 6 default looptimes */
-#define IBATINTERVAL (6 * 3500)       
+#define IBATINTERVAL (6 * 3500)
+#define GYRO_WATCHDOG_DELAY 100  // Watchdog for boards without interrupt for gyro
```

Comment:
use singline blank line between the new `#define` for `GYRO_WATCHDOG_DELAY ` and `IBATINTERVAL` since they are unrelated and improves readabilty.